ClosedXML/Excel/CalcEngine/Functions/Text.cs

```diff
@@ -192,7 +193,7 @@ private static string WildcardToRegex(string pattern)
 
         private static object Search(List<Expression> p)
         {
-            var search = WildcardToRegex((string)p[0]);
+            var search = WildcardToRegex(p[0]);
```

Comment:
Why did this line change? Did CodeMaid do this?

Reply:
yes, I ran code made on the files I had changed